test/e2e/support/utils.js

```diff
@@ -70,20 +70,36 @@ export async function newPost( postType ) {
 	await visitAdmin( 'post-new.php', postType ? 'post_type=' + postType : '' );
 }
 
+export async function getWindowWidth() {
+	return await page.evaluate(
+		() => {
+			return window.innerWidth;
+		}
+	);
+}
+
 export async function newDesktopBrowserPage() {
 	global.page = await browser.newPage();
+	await setDesktopViewPort();
+}
 
-	page.on( 'pageerror', ( error ) => {
-		// Disable reason: `jest/globals` doesn't include `fail`, but it is
-		// part of the global context supplied by the underlying Jasmine:
-		//
-		//  https://jasmine.github.io/api/3.0/global.html#fail
-
-		// eslint-disable-next-line no-undef
-		fail( error );
-	} );
+export async function newMobileBrowserPage() {
```

Comment:
Where is this function used?

Reply:
It was not being used I added it to be consistent we had setMobileViewport, setDesktopViewport, and newDesktopBrowserPage so it made sense to create newMobileBrowserPage. I removed it for now if it is required later it can be easily added.